README.md

```diff
@@ -323,6 +323,20 @@ Having a route with multiple parameters may affect negatively the performance, s
 
 **Note** that you must encode the parameters containing [reserved characters](https://www.rfc-editor.org/rfc/rfc3986#section-2.2).
 
+For [some use cases](https://github.com/delvedor/find-my-way/pull/211), you can customize the way the parameters are parsed by passing a custom decoding function.
```

Comment:
Please clarify what the use cases are instead of linking.

Reply:
done